packages/bff/src/utilities/errorMapper.ts

```diff
@@ -0,0 +1,21 @@
+import { constants } from "http2";
+import { ApiError, CommonErrorCodes } from "pagopa-interop-models";
+import { match } from "ts-pattern";
+import { ErrorCodes as LocalErrorCodes } from "../model/domain/errors.js";
+
+type ErrorCodes = LocalErrorCodes | CommonErrorCodes;
+
+const { HTTP_STATUS_INTERNAL_SERVER_ERROR, HTTP_STATUS_NOT_FOUND } = constants;
+
+export const getSelfcareErrorMapper = (error: ApiError<ErrorCodes>): number =>
+  match(error.code)
+    .with("selfcareEntityNotFilled", () => HTTP_STATUS_INTERNAL_SERVER_ERROR)
```

Comment:
```suggestion
```
Since it's a `500` we can just omit in order to let it go to the `otherwise`. Same for the following.